src/main/java/frc/robot/Systems/Shooter.java

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.Systems;
+
+import com.revrobotics.CANSparkBase;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.RelativeEncoder;
+import com.revrobotics.SparkPIDController;
+import com.revrobotics.CANSparkLowLevel.MotorType;
+
+import edu.wpi.first.wpilibj.motorcontrol.Spark;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants.RobotConstants;
+
+public class Shooter extends SubsystemBase {
+  
+  private CANSparkMax shooterTOP;
+  private CANSparkMax shooterBOTTOM;
+
+  private SparkPIDController shooterTopPID;
+  private SparkPIDController shooterBottomPID;
+
+  private RelativeEncoder shooterTopENC;
+  private RelativeEncoder shooterBottomENC;
+
+  /** Creates a new Shooter. */
+  public Shooter() {
+    shooterTOP = new CANSparkMax(RobotConstants.FLYWHEEL_TOP_ID, MotorType.kBrushless);
+    shooterBOTTOM = new CANSparkMax(RobotConstants.FLYWHEEL_BOTTOM_ID, MotorType.kBrushless);
+
+    //shooterTOP.restoreFactoryDefaults();
+    //shooterBOTTOM.restoreFactoryDefaults();
```

Comment:
I think I lied. This probably _is_ ok to do. I think what we don't want to do is call the `burnFlash` method, but even that isn't fully agreed upon on chief Delphi 🤷🏼